dapp_manager/storage.py

```diff
@@ -33,6 +33,14 @@ def save_pid(self, pid: int) -> None:
         with open(self.pid_file, "w") as f:
             f.write(str(pid))
 
+    def save_api_host(self, api_host: str) -> None:
+        with open(self.api_host_file, "w") as f:
```

Comment:
`self.api_host_file` is a `Path` object. Use `Path` api, instead of built-ins.
Same comment applies to other similar lines.